core/db/entities.py

```diff
@@ -373,6 +373,7 @@ class ProgrammeJunction(BaseModel):
         sqla.ForeignKey("submission_dim.id", ondelete="CASCADE"), nullable=False
     )
     programme_id: Mapped[GUID] = sqla.orm.mapped_column(sqla.ForeignKey("programme_dim.id"), nullable=False)
+    reporting_round = sqla.Column(sqla.Integer, nullable=True)
```

Comment:
Should this be non-nullable? Or is that a future PR change?

Reply:
Oh, I guess you're not populating it yet, so it has to be nullable

Reply:
Yep exactly